### PR TITLE
Closes #3130: Add descriptions for 6 AZ modules

### LINKED ISSUE #3130: Feature Request: Add Description to Quickstart Modules
## Motivation
Currently, administrators may encounter difficulties in understanding what each module does if there is no description in the .info.yml file for that module.

### Is your feature request related to a problem? Please describe.
No

## Proposed Resolution
Ensure all 53 Quickstart module `info.yml` files have a description in their info files. (This may need to be broken into multiple tickets if we can't agree on descriptions)

See search: https://github.com/search?q=repo%3Aaz-digital%2Faz_quickstart%20.info.yml&type=code

See drupal docs https://www.drupal.org/docs/develop/creating-modules/let-drupal-know-about-your-module-with-an-infoyml-file

### PR DESCRIPTION
**Pull request by BaoNguyen09**
## Description

Add descriptions for 6 AZ modules

## Related issues

Closes #3130 

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
   - [ ] Adding experimental module
   - [ ] Update experimental module
- **Minor release changes**
   - [x] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Upgrade experimental module to stable
   - [ ] Enable existing module by default or database update
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- **Other or unknown**
   - [x] Other or unknown

### Drupal core
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major or minor level update
- **Other or unknown**
   - [ ] Other or unknown

### Drupal contrib projects
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major level update
- **Other or unknown**
   - [ ] Other or unknown

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My change requires release notes.

### CODE REVIEW

File: modules/custom/az_flexible_block/az_flexible_block.info.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 name: 'Quickstart Flexible Block'
+description: "A content block with a title and seven dynamic block elements"
```
Comment (camikazegreen) [inline modules/custom/az_flexible_block/az_flexible_block.info.yml:2]:
```suggestion
description: "A content block with a title and dynamic block elements"
```
Less specificity here will make this easier to maintain long term.

File: modules/custom/az_google_tag/az_google_tag.info.yml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 name: 'Quickstart Google Tag Manager'
+description: "Allows to configure the Institution-wide Google Tag Manager container (AZ Comprehensive)"
```
Comment (camikazegreen) [inline modules/custom/az_google_tag/az_google_tag.info.yml:2]:
```suggestion
description: "Allows configuration of the institution-wide Google Tag Manager container (AZ Comprehensive)"
```

File: modules/custom/az_seo/az_seo.info.yml [hunk 3]
```diff
@@ -1,4 +1,5 @@
 name: 'Quickstart SEO'
+description: "Provides default configuration for modules to help implementing meta data for search engine optimization and track website traffic using Google Analytics"
```
Comment (camikazegreen) [inline modules/custom/az_seo/az_seo.info.yml:2]:
```suggestion
description: "Provides default configuration for modules to help implement meta data for search engine optimization and track website traffic using Google Analytics"
```

File: modules/custom/az_select_menu/az_select_menu.info.yml [hunk 4]
```diff
@@ -1,4 +1,5 @@
 name: 'Quickstart Select Menu Block'
+description: "Allows to display a menu as a dropdown selection, also known as persona menu"
```
Comment (camikazegreen) [inline modules/custom/az_select_menu/az_select_menu.info.yml:2]:
```suggestion
description: "Allows displaying a menu as a dropdown selection, also known as persona menu"
```

**Review by camikazegreen — DISMISSED** [review]

**Review by trackleft — APPROVED** [review]

File: modules/custom/az_seo/az_seo.info.yml [hunk 5]
```diff
@@ -1,4 +1,5 @@
 name: 'Quickstart SEO'
+description: "Provides default configuration for modules to help implement meta data for search engine optimization and track website traffic using Google Analytics"
```
Comment (trackleft) [inline modules/custom/az_seo/az_seo.info.yml:2]:
```suggestion
description: "Provides default configuration for modules to help implement meta data for search engine optimization and track website traffic."
```

**Review by joeparsons — APPROVED** [review]

**Review by trackleft — APPROVED** [review]

**Review by trackleft — APPROVED** [review]

File: modules/custom/az_block_types/az_block_types_quick_links/az_block_types_quick_links.info.yml [hunk 6]
```diff
@@ -1,4 +1,5 @@
 name: 'Quickstart Block Types - Quick Links'
+description: "An embeddable content block with block fields that accept links"
```
[the remark this answers is not in the document]
Comment (bberndt-uaz) [reply modules/custom/az_block_types/az_block_types_quick_links/az_block_types_quick_links.info.yml:2]: It looks like this still needs a period at the end.

```suggestion
description: "An embeddable content block with block fields that accept links."
```

**Review by joeparsons — APPROVED** [review]

**Review by bberndt-uaz — APPROVED** [review]

**Pull request merged**